LayoutTests/imported/w3c/web-platform-tests/html/semantics/interactive-elements/the-dialog-element/focus-previous-iframe.tentative-expected.txt

```diff
@@ -1,3 +1,4 @@
+CONSOLE MESSAGE: Error: assert_equals: expected Element node <input></input> but got Element node <iframe srcdoc="<input><dialog> Dialog in child </dialog>...
```

Comment:
I confirmed that this is also in Chromium: https://source.chromium.org/chromium/chromium/src/+/main:third_party/blink/web_tests/external/wpt/html/semantics/interactive-elements/the-dialog-element/focus-previous-iframe.tentative-expected.txt

so probably a test issue.